napari_skimage_regionprops/_regionprops.py

```diff
@@ -184,3 +196,371 @@ def ellipsoid_axis_lengths(table):
 
 regionprops_table_all_frames = analyze_all_frames(regionprops_table)
 register_function(regionprops_table_all_frames, menu="Measurement > Regionprops of all frames (nsr)")
+
+
+def make_element_wise_dict(list_of_keys, list_of_values):
+    """Return an element-wise dictionary of two lists.
+
+    From a list of keys and a list of values, return a dictionnary linking
+    those lists. Both lists must have the same length.
+
+    Parameters
+    ----------
+    list_of_keys : List
+        list to be used as dictionnary keys.
+    list_of_values : List
+        list to be used as dictionnary values.
+
+    Returns
+    -------
+    Dictionnary
+    """
+    return dict(
+                list(
+                    map(
+                        lambda list_of_keys, list_of_values:
+                        [list_of_keys, list_of_values],
+                        *[list_of_keys, list_of_values]
+                        )
+                    )
+                )
```

Comment:
```suggestion
    return {k: v for k, v in zip(list_of_keys, list_of_values)}
```
I presume this is doing the same. Am I right?

Reply:
Yes! Much simpler and more elegant, thanks!